src/main/java/de/qabel/desktop/ui/remotefs/RemoteFSController.java

```diff
@@ -514,7 +514,7 @@ private void deleteItem(TreeItem<BoxObject> item) {
 
                 Alert alert = new Alert(Alert.AlertType.CONFIRMATION);
                 alert.setTitle(resourceBundle.getString("remoteFsDeleteQuestion"));
-                alert.setHeaderText(resourceBundle.getString("remoteFsDeleteFolder") + item.getValue().getName() + " ?");
+                alert.setHeaderText(resourceBundle.getString("remoteFsDeleteFolder ") + item.getValue().getName() + " ?");
```

Comment:
the whitespace should be outside the getString:
`alert.setHeaderText(resourceBundle.getString("remoteFsDeleteFolder")+ " " + item.getValue().getName() + " ?"); `